resolwe_bio/processes/alignment/star.py

```diff
@@ -0,0 +1,704 @@
+            unmaped_out_1 = f"{mate1_name}_unmapped.out.mate1.fastq"
+            star_unmapped_r1.rename(unmaped_out_1)
+
+            return_code, _, _ = Cmd["pigz"][unmaped_out_1] & TEE(retcode=None)
+            if return_code:
+                self.error("Compression of unmapped mate 1 reads failed.")
+
+            outputs.unmapped_1 = f"{unmaped_out_1}.gz"
+
+        star_unmapped_r2 = Path("Unmapped.out.mate1")
+        if (
+            inputs.reads.type.startswith("data:reads:fastq:paired:")
+            and star_unmapped_r2.is_file()
+        ):
+            unmaped_out_2 = f"{mate2_name}_unmapped.out.mate2.fastq"
+            star_unmapped_r2.rename(unmaped_out_2)
+
+            return_code, _, _ = Cmd["pigz"][unmaped_out_2] & TEE(retcode=None)
+            if return_code:
+                self.error("Compression of unmapped mate 2 reads failed.")
+            outputs.unmapped_2 = f"{unmaped_out_2}.gz"
+
+        self.progress(0.8)
+
+        aligned_reads = (
+            "Aligned.sortedByCoord.out.bam" if inputs.star_sort else "Aligned.out.bam"
+        )
+        out_bam = f"{mate1_name}.bam"
+        out_bai = f"{out_bam}.bai"
+
+        sort_params = [
+            aligned_reads,
+            "-o",
+            out_bam,
+            "-@",
+            self.requirements.resources.cores,
+        ]
+        return_code, _, _ = Cmd["samtools"]["sort"][sort_params] & TEE(retcode=None)
+        if return_code:
+            self.error("Samtools sort command failed.")
+
+        outputs.bam = out_bam
+
+        return_code, _, _ = Cmd["samtools"]["index"][out_bam, out_bai] & TEE(
+            retcode=None
+        )
+        if return_code:
+            self.error("Samtools index command failed.")
+
+        outputs.bai = out_bai
+
+        self.progress(0.9)
+
+        if inputs.detect_chimeric.chimeric:
+            out_chimeric = f"{mate1_name}_chimeric.out.sam"
+            Path("Chimeric.out.sam").rename(out_chimeric)
+            outputs.chimeric = out_chimeric
+
+        if inputs.t_coordinates.quant_mode:
+            out_transcriptome = f"{mate1_name}_aligned.toTranscriptome.out.bam"
+            Path("Aligned.toTranscriptome.out.bam").rename(out_transcriptome)
+            outputs.alignment_transcriptome = out_transcriptome
+
+        out_stats = f"{mate1_name}_stats.txt"
+        Path("Log.final.out").rename(out_stats)
+        outputs.stats = out_stats
+
+        out_sj = f"{mate1_name}_SJ.out.tab"
+        Path("SJ.out.tab").rename(out_sj)
+        outputs.sj = out_sj
+
+        outputs.species = inputs.genome.output.species
+        outputs.build = inputs.genome.output.build
+
+        bamtobigwig_params = [
+            out_bam,
+            inputs.genome.output.species,
+            self.requirements.resources.cores,
+            inputs.output_options.tool_bigwig,
+            inputs.output_options.bin_size_bigwig,
+        ]
+        check_bigwig, _, _ = Cmd["bamtobigwig.sh"][bamtobigwig_params] & TEE(
```

Comment:
Perhaps add a comment that this process outputs to `outputs.bigwig` output field.